.github/workflows/tests.yml

```diff
@@ -122,7 +126,7 @@ jobs:
       run: |
         go install github.com/jackc/tern@latest
         tern migrate -m "$TERN_MIGRATIONS_DIR"
-        make build
+        make image-builder-db-test
         ./image-builder-db-test
```

Comment:
Only build what is needed for the test not all binaries. They are built elsewhere now.